<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Mohamed-Saeed" failures="0" tests="4" name="tests.UserRegistrationWithDDTAndDataProvider" time="4.687" errors="1" timestamp="2023-10-28T20:16:00 EET" skipped="1">
  <testcase classname="tests.UserRegistrationWithDDTAndDataProvider" name="RegisteredUserCanLogin" time="0.004">
    <error message="
[public void tests.UserRegistrationWithDDTAndDataProvider.RegisteredUserCanLogin(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: RegisteredUserCanLogin([java.lang.String arg0, java.lang.String arg1])
Arguments: [(java.lang.String) Saz,(java.lang.String) fort,(java.lang.String) testSaz016@gmail.com,(java.lang.String) 123456]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void tests.UserRegistrationWithDDTAndDataProvider.RegisteredUserCanLogin(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: RegisteredUserCanLogin([java.lang.String arg0, java.lang.String arg1])
Arguments: [(java.lang.String) Saz,(java.lang.String) fort,(java.lang.String) testSaz016@gmail.com,(java.lang.String) 123456]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:909)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- RegisteredUserCanLogin -->
  <system-out/>
  <testcase classname="tests.UserRegistrationWithDDTAndDataProvider" name="RegisteredUserCanLogout" time="0.001">
    <skipped/>
  </testcase> <!-- RegisteredUserCanLogout -->
  <system-out/>
  <testcase classname="tests.UserRegistrationWithDDTAndDataProvider" name="UserCanRegisterSuccssfully" time="1.757"/>
  <system-out/>
  <testcase classname="tests.UserRegistrationWithDDTAndDataProvider" name="UserCanRegisterSuccssfully" time="2.925"/>
  <system-out/>
</testsuite> <!-- tests.UserRegistrationWithDDTAndDataProvider -->
